WordPress/Resources/release_notes.txt

```diff
@@ -1,10 +1,3 @@
-* [*] Remove large title in Reader and Notifications tabs. [#20271]
-* [*] Reader: Change the following button cog icon. [#20274]
-* [*] Change the Reader's navigation bar background color to match other screens. [#20278]
-* [*] Tweak My Site Dashboard Cards UI. [#20303]
-* [*] [internal] Refactored the Core Data operations (saving the site data) after a new site is created. [#20270]
-* [*] [internal] Refactored updating user role in the "People" screen on the "My Sites" tab. [#20244]
-* [*] [internal] Refactor managing social connections and social buttons in the "Sharing" screen. [#20265]
-* [*] [internal] Refactor uploading media assets. [#20294]
-* [*] Block editor: Allow new block transforms for most blocks. [https://github.com/WordPress/gutenberg/pull/48792]
-* [*] Visual improvements were made to the in-app survey along with updated text to differentiate between the WordPress and Jetpack apps. [#20276]
+We don’t know about you, but we’re feeling 22. In honor of the occasion (and because you’ve been asking for it), you can now transform most types of blocks into other block types, like quotes, columns, and groups. Nice.
+
+You’ll also see a few small visual changes to the My Site dashboard and the in-app survey.
```

Comment:
The original copy included the "We made some design adjustments to the Reader" paragraph. I decided to remove it because new users don't have access to Reader or Notifications.